docs/operate/reference/advanced-modules/_index.md

```diff
@@ -46,6 +46,73 @@ Running {{< glossary_tooltip term_id="modular-resource" text="modular resources"
 
 However, if you are unable to use modular resources because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/operate/reference/advanced-modules/custom-components-remotes/) on your machine.
 
+## Package and deploy using Docker
+
+In rare cases, you may need to package and deploy a module using Docker.
```

Comment:
Be aware that we are not providing complete info here. We're showing how to pull and run a container. We will likely need to look into how to package a module with Docker. So far this really only covers the deploy aspect.

Please file a follow up ticket to cover this. A full example for reference is really what I think we need here.

Reply:
https://viam.atlassian.net/browse/DOCS-3863